app/controllers/api/events_controller.rb

```diff
@@ -0,0 +1,57 @@
+# rubocop:disable Style/ClassAndModuleChildren
+# :reek:InstanceVariableAssumption
+
+class Api::EventsController < ApplicationController
```

Comment:
what about adding folder v1? so we will have Api::V1::EventsController. It will allow to switch between API versions in future